README.md

```diff
@@ -1,31 +1,31 @@
 # Contentful UI Extensions SDK
 
 The UI Extensions SDK allows you to customize and extend the functionality of [Contentful](https://www.contentful.com)
-Web Application's entry editor. The editor itself is a container for many "widgets" that enable editors to manipulate the content stored in content fields. Widgets' complexity varies. They can be simple user interface controls, such as a dropdown, or more complex micro web applications such as our Markdown editor. They are decoupled entities from field types, and can be reused (for example using a dropdown widget to edit number or text fields).
+Web Application's entry editor. The editor itself is a container for many components that enable editors to manipulate the content stored in content fields. Extensions' complexity varies. They can be simple user interface controls, such as a dropdown, or more complex micro web applications such as our Markdown editor. They are decoupled entities from field types, and can be reused (for example using a dropdown to edit number or text fields).
 
-Previously, the Contentful Web Application only offered our core platform widgets as options to manipulate fields' content. Now, with the UI Extensions SDK it is possible to create custom widgets to further personalize your users' needs.
-Core widgets and custom widgets are both built on top of the same API, making them virtually identical in functionality with the main difference being that custom widgets are rendered inside a secure iframe. The next step in our roadmap is to open source our core widgets to make it even easier to build custom ones.
+Previously, the Contentful Web Application only offered our core platform components as options to manipulate fields' content. Now, with the UI Extensions SDK it is possible to create custom extensions to further personalize your users' needs.
+Core components and custom extensions are both built on top of the same API, making them virtually identical in functionality with the main difference being that custom extensions are rendered inside a secure iframe. The next step in our roadmap is to open source our core components to make it even easier to build custom ones.
```

Comment:
"virtually identical in functionality" -> This implies that components and extensions are identical in functionality and they are not.